jcsv.go

```diff
@@ -1,11 +1,84 @@
 package jcsv
 
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+)
+
 func JsonToCsv(j []byte, addHeaders bool) ([]byte, error) {
-	// TODO: convert JSON data in `j` into CSV format and return
-	return nil, nil
+	//create file object
+	var JSON file
+
+	//convert data in j to Data
+	json.Unmarshal(j, &JSON.Data)
```

Comment:
The user may have provided an invalid `JSON`. Unmarshal may fail. You should handle the errors.

Reply:
all right, I will resolve this issue.